### PR TITLE
Delete not empty edition

### PR DESCRIPTION
**Pull request by ruvmello**
A toast is now shown when an attempt to delete a not empty edition is made.

### CODE REVIEW

File: frontend/src/components/edition/editionRowComponent.tsx [hunk 1]
```diff
@@ -38,8 +39,13 @@ export function EditionRowComponent(props: EditionProps) {
     }
 
     async function deleteEdition() {
-        const response = await editionDelete(edition._links.self.href);
-        if (response.status == StatusCodes.NO_CONTENT) {
+        const response = await editionDelete(edition._links.self.href).catch((request) => {
```
Comment (jitsedesmet) [inline frontend/src/components/edition/editionRowComponent.tsx:42]: using catch here with a callback, in combination with async isn't very conssistent.

**Review by jitsedesmet — COMMENTED** [review]
Almost :)

File: frontend/src/components/edition/editionRowComponent.tsx (hunk 1 above)
Comment (ruvmello) [reply frontend/src/components/edition/editionRowComponent.tsx:42]: Well can you give me another way? I simply tried checking the status in an else if, but the problem is that when the status code is 409 it is considered an error, that's why I need the catch

Comment (ruvmello) [reply frontend/src/components/edition/editionRowComponent.tsx:42]: I will try using try and catch

**Review by jitsedesmet — APPROVED** [review]

**Pull request merged**